index.html

```diff
@@ -424,36 +424,123 @@ <h4>Double-sided ruby, when both read aloud</h4>
+	      non-particles in ruby annotations may well be mistakenly
+	      interpreted as particles.  Consequently, such occurrences
+	      of <span lang="ja">は</span> and <span lang="ja">へ</span>
+	      may well be mistakenly read aloud as /wa/ and /e/, respectively.</p>
+
+	    <p>For example, consider  
+	      <span lang="ja">やがて<ruby><rb>廃止</rb><rt>はいし</rt></ruby>になる</span>.
+	      This sentence means "It will be abolished eventually".  But
+	      if <span lang="ja">やがてはいしになる</span> is passed to the
+	      TTS engine, <span lang="ja">は</span> may well be mistakenly
+	      read aloud as /wa/ rather than /ha/.  The result means
+	      "I will eventually become a doctor".
+	    </p>
+
+	    <p>Here are some similar examples.  All occurrences of
+	      <span lang="ja">は</span> and <span lang="ja">へ</span>
+	      in ruby annotations are likely to be mistakenly read aloud.</p>
+	    
+	    <ul>
+	      <li><span lang="ja">人員<ruby><rb>配置</rb><rt>はいち</rt></ruby></span> </li>
+	      <li><span lang="ja">自然<ruby><rb>破壊</rb><rt>はかい</rt></ruby></span> </li>
+	      <li><span lang="ja">社会<ruby><rb>波紋</rb><rt>はもん</rt></ruby></span> </li>
+	      <li><span lang="ja">天皇<ruby><rb>陛下</rb><rt>へいか</rt></ruby></span></li>
+	      <li><span lang="ja">大学<ruby><rb>併願</rb><rt>へいがん</ruby></span></li>
+	      <li><span lang="ja">学級<ruby><rb>閉鎖</rb><rt>へいさ</rt></ruby></span></li>
+	    </ul>
+	  </section>
+	  <section>
+	    <h5>Inconsistency between the first and subsequent occurrences</h5>
+            <p>
+	      As described in <a href="#furigana-background"><span class="secno">2.1</span></a>,
+	      furigana as a ruby annotation may be attached to only the
+	      first occurrence of a CJK ideographic character or a word
+	      composed from such characters.  Thus, there is a risk that
+	      the first occurrence and the others are read aloud differently.
+	      For example, consider <span lang="ja">智子</span> as the name
+	      of a character in the novel. There are two possible readings
+	      of this name: <span lang="ja">さとこ</span> and
+	      <span lang="ja">ともこ</span>. If <span lang="ja">さとこ</span>
+	      as a ruby annotation is attached only to the first occurrence
+	      of the name, it will be read as <span lang="ja">さとこ</span>
+	      and the other occurrences may be read as <span lang="ja">ともこ</span>.
+	      The reader would then think that <span lang="ja">さとこ</span>
+	      and <span lang="ja">ともこ</span> are different characters.</p>
           <aside class="note" title="" id="n20211101002">
-            One way to avoid this problem is for the text-to-speech engine to maintain a correspondence table between ruby bases and ruby annotations.
+            One approach to avoid this problem is to create a table of
+	    ruby base-annotation pairs. When a CJK ideographic character
+	    or a word composed of such characters is encountered, this
+	    table allows the TTS engine to receive the ruby annotation
```

Comment:
just a comment..
I am still not sure whether exact match (by means base text of `<ruby>` ruby block is identical to the table) or full text match to `<rb>`-s is better...

Reply:
I would like to be silent about that here.  Although I mentioned this approach (which is actually proposed in a project supported by the JP government), I do not believe that it will be implemented.  PLS looks much better to me.

Reply:
I totally agree with PLS, but at the same time similar issue exists - we need to define how grapheme would work in Japanese text for replacement of ruby feature...

Reply:
> I totally agree with PLS, but at the same time similar issue exists - we need to define how grapheme would work in Japanese text for replacement of ruby feature...

True.  This is a issue of PLS.  Not this note.